docs/examples_api_basic.rst

```diff
@@ -2,24 +2,13 @@
 Examples - API: Basic
 =====================================================
 
-.. Generic examples
```

Comment:
got a little carried away in here and decided to move the pulse examples below pi-top 4 & pma ones, and standardise the section header comments...

Reply:
great!